MEMAnalyzer_Backend/Business/Constants/Roles.cs

```diff
@@ -4,5 +4,6 @@ public static class Roles
     {
         public const string USER = "User";
```

Comment:
ываыв

- [ ываывыав] 
```suggestion
        public const string USER = "User";
```

```suggestion
        public const string USER = "User";
```